lib/src/common/model/changes.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'changes.g.dart';
+
+@JsonSerializable()
+class Ref {
+  Ref(this.from);
```

Comment:
Can we mark these as const?

A good follow up would be to add https://dart.dev/tools/linter-rules/prefer_const_constructors to the lints in this package

Reply:
Yes sir! I will go ahead and do both!

Reply:
Hmm, it seems to already be there in the analyzis_options.yaml file.